transcom/mymove
# MAIN B-20488 - Services Counselor Order Upload

## [B-20488](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-20488)

INT PR: https://github.com/transcom/mymove/pull/13273

## Summary

This is to allow SC to view/edit orders until the TOO approves move/shipment(sent to prime).

- Added missing 'NEW' tag to SC that shows up when amended order has been uploaded(currently only shows for TOO).
- SC can view/edit after they submit the move to TOO.
- SC cannot view/edit once TOO approves the move/shipment.
- Same with allowances since they seem to be bundled with orders.

### How to test

1. As a customer -> create a move with HHG shipment and finish submitting.
2. As a SC -> go to the move -> edit orders information -> submit the move and you should still be able to view/edit orders after. Same with view/edit allowances button.
3. AS a TOO -> go to the move -> approve the shipment and send to prime.
4. As a SC, use the Search Move tab to find the move -> verify that view/edit orders and allowance button is not visible.

## Screenshots
![image](https://github.com/user-attachments/assets/1c1071ac-6e7f-41ad-bde1-6063e0815105)
![image](https://github.com/user-attachments/assets/97ef41bb-5944-43e3-acde-48dd605525ea)
